ci/build_docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e -E -u -o pipefail
+
+rapids-print-env
+
+make -C docs html
```

Comment:
This script is pretty thin right now, but it'll have more added to it in #129. See the diff there for details.

Publishing a CPU-only conda package will allow us to run docs builds on a machine that doesn't have a GPU 😁